modules/publicize/publicize.php

```diff
@@ -97,18 +97,15 @@ function get_profile_link( $service_name, $c ) {
 		$cmeta = $this->get_connection_meta( $c );
 
 		if ( isset( $cmeta['connection_data']['meta']['link'] ) ) {
-			if ( 'facebook' == $service_name && 0 === strpos( parse_url( $cmeta['connection_data']['meta']['link'], PHP_URL_PATH ), '/app_scoped_user_id/' ) ) {
```

Comment:
The changes in #1944 are the preferred version in both environments. I'll plan on updating this in WPCOM later today. Thanks for catching this @jeherve.